chess-AI/experimental-work/mcts.py

```diff
@@ -1,73 +1,161 @@
-#https://web.stanford.edu/~surag/posts/alphazero.html
+from math import sqrt
+import numpy as np
+from state_representation import get_cnn_input
 
 class Mcts:
-    def search(s,game,nnet):
 
```

Comment:
Should have [docstring](https://google.github.io/styleguide/pyguide.html#384-classes) describing the class here.